src/Octoshift/GithubClient.cs

```diff
@@ -111,27 +114,27 @@ public virtual async IAsyncEnumerable<JToken> GetAllAsync(string url)
         public virtual async Task<string> DeleteAsync(string url) => (await SendAsync(HttpMethod.Delete, url)).Content;
 
         private async Task<(string Content, KeyValuePair<string, IEnumerable<string>>[] ResponseHeaders)> SendAsync(
-            HttpMethod httpMethod, string url, object body = null, HttpStatusCode status = HttpStatusCode.OK)
+            HttpMethod httpMethod, string url, object body = null, HttpStatusCode status = HttpStatusCode.OK, string acceptHeader = null)
         {
             url = url?.Replace(" ", "%20");
-
             _log.LogVerbose($"HTTP {httpMethod}: {url}");
 
-            if (body != null)
+            using var request = new HttpRequestMessage(httpMethod, new Uri(url));
```

Comment:
I would follow `HttpClient`'s implementation and use the following method to create the `Uri`:
```c#
private Uri CreateUri(string? uri) => string.IsNullOrEmpty(uri) ? null : new Uri(uri, UriKind.RelativeOrAbsolute);
```

Reply:
If you get the latest main, this method is already rewritten.